demisto_client/demisto_api/rest.py

```diff
@@ -220,7 +220,10 @@ def request(self, method, url, query_params=None, headers=None,
             # In the python 3, the response.data is bytes.
             # we need to decode it to string.
             if six.PY3 and r.getheader("Content-Type") != "application/octet-stream":
-                r.data = r.data.decode('utf8')
+                try:
+                    r.data = r.data.decode('utf8')
+                except UnicodeDecodeError:
+                    pass
```

Comment:
@MosheEichler What string do we get here? any non-unicode surprises? for documentation and in case this ever happens again

Reply:
I sent you a file in slack of what it returns

Suggested solution:
Maybe we can use str() instead of decode().
str() does not return an error.

Reply:
`str` is the [py3 equivalent of `decode`](https://stackoverflow.com/a/14473492), and the `if six.PY3` above makes sure we're on py3. @DeanArbel wdyt?